polypheny/__init__.py

```diff
@@ -1,66 +1,71 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-#
-# Copyright 2019-2021 The Polypheny Project
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-# 
-#  http://www.apache.org/licenses/LICENSE-2.0
-# 
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
+from polypheny.connection import Connection, Cursor
+from polypheny.exceptions import *
 
-# Python Db API v2
-# https://www.python.org/dev/peps/pep-0249/
+import datetime
+from typing import Union, Tuple
 
+apilevel = '2.0'
```

Comment:
This should not be hardcoded. Use prism api major and minor version or remove variable if not used.

Reply:
This is not the prism API level but the Python Database API level.  See [PEP 249](https://peps.python.org/pep-0249/#apilevel).